person.rb

```diff
@@ -17,8 +17,8 @@ def initialize(age:, name: 'Unknown', parent_permission: true)
     @rentals = []
   end
 
-  def add_rental(rental)
-    @rentals << rental
+  def add_rental(date, book)
+    @rentals << Rental.new(date, book, self)
```

Comment:
*[OPTIONAL]*

- Hey, I think you don't really need to push to the @rentals instance variable from here, you'd add it later so that this doesn't confuse you later in other milestones. Also, in Rentals class, you've already pushed into those arrays `books, person`, You could leave it this way:

`Rentals.new(date, book, self)`